docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -17,16 +17,60 @@ A good example notebook exhibits the following attributes:
    Avoids abstraction traps.
    See `Selbst et al. (2020) <https://andrewselbst.files.wordpress.com/2019/10/selbst-et-al-fairness-and-abstraction-in-sociotechnical-systems.pdf>`_.
 4. **Substantiated**: Discusses trade-offs and compares alternatives.
-   Describes why using particular Fairlearn functionalities makes sense.
+   Describes why using particular Fairlearn functionalities makes sense if
```

Comment:
This sentence is a bit tautological.

Reply:
Maybe something like - "Describes why using particular Fairlearn functionalities make sense for this task or use case." ?

Reply:
But we may not be using Fairlearn in every use case (?)